lib/tasks/sdk.rake

```diff
@@ -39,6 +39,8 @@ task 'setup_env' do
   `cp #{gem_home}/lib/config/datadog.conf #{ENV['SDK_HOME']}/embedded/dd-agent/datadog.conf`
   # This sometimes causes check setup to fail
   FileUtils.rm Dir.glob('setuptools*.zip')
+  # Download JMX Fetch, too
+  Rake::Task['setup_libs'].invoke
```

Comment:
I think I'm fine with this, but we must bear in mind that the jmxfetch version will vary across branches, so once you checkout a different branch, you need to `setup_libs` again.